resources/sdk/clisdkclient/templates/root.mustache

```diff
@@ -12,16 +12,29 @@ import (
 	"flag"
 	"fmt"
 	"strings"
+	"encoding/json"
+    "io/ioutil"
+    "net/http"
+    "time"
 	"github.com/mypurecloud/platform-client-sdk-cli/build/gc/cmd/dummy_command"
 	{{addImports}}
 	"os"
 )
 
+type LatestVersion struct {
+	Version string `json:"version"`
+}
+
 var (
 	profileName string
 	output string
 )
 
+var (
```

Comment:
These variable declarations can be declared in the var block above. Usually the style is to have a single var block per file